README.md

```diff
@@ -6,9 +6,6 @@
 [![codecov](https://codecov.io/gh/stac-utils/stac-task/branch/main/graph/badge.svg)](https://codecov.io/gh/stac-utils/stac-task)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-
-
-
 This Python library consists of the Task class, which is used to create custom tasks based
 on a "STAC In, STAC Out" approach. The Task class acts as wrapper around custom code and provides
```

Comment:
When I first learned about cirrus and tasks, the biggest confuser was around tasks being "wrappers"; I thought that wrappers meant around the compute (docker or lambda), e.g. this was code that was used to somehow invoke docker containers, not be executed from within a compute context.

I don't know the best place to talk about this; also happy to expand on what I mean.